README.md

```diff
@@ -1 +1,117 @@
-# sound_manager
+# ad_sound_manager
+
+## Overview
+This node switches the sound to play depending on the state of the system.
```

Comment:
```suggestion
Any autonomous driving vehicle have to notify its action to surrounding people. This node selects both a recorded voice and BGM depending on a state of whole system which combined  the Autoware and a vehicle.
```

Reply:
I closed this thread because of [this comment](https://github.com/eve-autonomy/ad_sound_manager/pull/1#discussion_r895536877).